.github/workflows/deploy-gcp-tests.yml

```diff
@@ -432,6 +432,8 @@ jobs:
           --ssh-flag="-o ConnectTimeout=5" \
           --command \
           "\
+          # Wait for the disk to be attached
+          while [[ ! -e /dev/sdb ]]; do sleep 1; done && \
```

Comment:
Optional: Can we do `set -x` at the start of these scripts, so if there is a hang, we can see which script line it is on?

Reply:
I'll apply this later on another PR